databases/relational-databases.md

```diff
@@ -118,13 +122,15 @@ Make sure you don't slow down a database for other users. This can happen if you
 Because you must define a schema to store data in a relational database, some developers think that they are harder to use than "schema-less" databases like MongoDB. However, taking the effort to use a schema for your data can drastically reduce the likelihood of bugs as your project matures, so many developers choose relational DBs.
 
 ### Guided Practice
+
 - Install Postgres locally
 - Connect to Postgres with the commandline client, then:
   - Create a _database_
   - Create a _table_
   - Create another table that is different but related, and that shares at least one column with the first table created (i.e. create a table that can be logically `JOIN`ed to the first table)
   - Add rows of data to the new table(s)
   - Execute a few queries to retrieve data, as guided by instructor or volunteers. (While this will likely be review of content from the [SQL](https://github.com/Techtonica/curriculum/blob/master/SQL/sql.md) lesson plan, practice will help sharpen these skills.)
+Go through the course [ Databases Course: Querying Relational Databases (145 min)](https://teamtreehouse.com/library/querying-relational-databases). In this course the concepts of Set Theory and Database Normalization are introduced. You will also learn about Database Keys, without which database would not be able to function and Table Relationships, which help us to describe the way that one tables links or relates to another.
```

Comment:
Line 133 needs to be a bullet point- look at it on the markdown preview. Please split this last sentence into two... it’s too long.

Also change one tables to one table.

Reply:
made the changes.